modules/ROOT/pages/am-permissions.adoc

```diff
@@ -3,15 +3,20 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-The *Access Management* section of Anypoint Platform enables you (as an Organization Administrator) to control user access to dashboards, reports, and metrics by providing two predefined product-specific permissions:
+Anypoint Monitoring has two product-specific permissions that control access to assets within Anypoint Monitoring:
```

Comment:
Suggestion to consider:  Anypoint Monitoring has two product-specific permissions that control access to its assets:

Reply:
```suggestion
Anypoint Monitoring has two product-specific permissions that control access to its assets:
```